cfpb/sbl-frontend
# development(yarn update): switch back to initial git branch at end of the script

closes #351 

## Changes
- development(yarn update): Switch back to the git branch from which `yarn update` was run in

## How to test this PR

1. Be in any branch other than `main`
2. `yarn update`
3. Check to see if your current branch is the one where you ran `yarn update` instead of `main`.
